src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XPathNavigatorExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+using System.Xml.XPath;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    internal static class XPathNavigatorExtensions
+    {
+        internal static XPathNavigator SelectFirstChild(this XPathNavigator navigator, string name)
+        {
+            return navigator.SelectChildren(name, "")
```

Comment:
For readability I think it would be useful to name a constant `const string EmptyNamespace = ""` and then use that in the three places, as otherwise I wasn't sure what it signified just from the diff and had to go and look it up.